src/material/dialog/dialog-config.ts

```diff
@@ -104,6 +104,9 @@ export class MatDialogConfig<D = any> {
   /** Aria label to assign to the dialog element. */
   ariaLabel?: string | null = null;
 
+  /** Whether this a modal dialog. Used to set the `aria-modal` attribute. */
```

Comment:
```suggestion
  /** Whether this is a modal dialog. Used to set the `aria-modal` attribute. */
```

(also for the bottom sheet)